quantity-pickers/src/main/java/com/decathlon/vitamin/compose/quantity/pickers/VitaminQuantityPickers.kt

```diff
@@ -69,59 +75,98 @@ object VitaminQuantityPickers {
+                        ),
+                        onClick = onSubtractClicked
                     )
+
+                    MiddleTextField(
+                        modifier = Modifier
+                            .background(
+                                color = colors.backgroundColor.copy(
+                                    alpha = if (!addEnabled && !subtractEnabled) {
+                                        VtmnStatesDisabled
+                                    } else VtmnStatesEnabled
+                                )
+                            )
+                            .fillMaxHeight()
+                            .weight(1.0f),
+                        addEnabled = addEnabled,
+                        subtractEnabled = subtractEnabled,
+                        colors = colors,
+                        textStyle = textStyle,
+                        value = value,
+                        onValueChange = onValueChange,
+                        editTextEnabled = editTextEnabled,
+                        keyboardOptions = keyboardOptions,
+                        keyboardActions = keyboardActions
+                    )
+
+                    IconButton(
+                        painter = painterResource(id = R.drawable.ic_vtmn_add_fill),
+                        contentDescription = stringResource(id = R.string.vtmn_add_button_description),
+                        enabled = addEnabled,
+                        colors = colors,
+                        shape = shape.copy(
+                            topStart = CornerSize(0.dp),
+                            bottomStart = CornerSize(0.dp)
+                        ),
+                        onClick = onAddClicked
+                    )
+                }
+                ErrorLabel(errorMessage = errorMessage)
+            }
+        }
+    }
+
+    @Composable
+    private fun ErrorLabel(
+        errorMessage: String?
+    ) {
+        errorMessage?.let {
+            Box(
+                modifier = Modifier
+                    .size(4.dp)
+            )
+            Row(
+                verticalAlignment = Alignment.CenterVertically
             ) {
-                IconButton(
-                    painter = painterResource(id = R.drawable.ic_vtmn_subtract_fill),
-                    contentDescription = stringResource(id = R.string.vtmn_subtract_button_description),
-                    enabled = subtractEnabled,
-                    colors = colors,
-                    shape = shape.copy(
-                        topEnd = CornerSize(0.dp),
-                        bottomEnd = CornerSize(0.dp)
-                    ),
-                    onClick = onSubtractClicked
+                Icon(
+                    modifier = Modifier
+                        .size(16.dp),
+                    painter = painterResource(id = R.drawable.ic_vtmn_information_line),
+                    contentDescription = stringResource(id = R.string.vtmn_information_icon_description),
```

Comment:
Not sure we need this content description for the icon as we already have the error text just after. Maybe we can ask @GaspardMathon or @Sabrinavigil their opinion on this.

Reply:
Yes could be interesting to have this semantic error at QuantityPicker Level. 
It should be more useful to have an error / warning information info when you are focused on the component to update it